Fix for dims.hpp:25:28: error: int64_t was not declared in this scope
Bug: https://bugs.gentoo.org/922778
Upstream fix: https://github.com/oneapi-src/oneDNN/pull/1792
--- a/tests/benchdnn/utils/dims.hpp
+++ b/tests/benchdnn/utils/dims.hpp
@@ -18,6 +18,7 @@
 #define UTILS_DIMS_T_HPP
 
 #include <cassert>
+#include <cstdint>
 #include <iostream>
 #include <string>
 #include <vector>
